model/http/spans.yaml

```diff
@@ -45,6 +45,8 @@ groups:
         requirement_level: opt_in
       - ref: http.response.body.size
         requirement_level: opt_in
+      - ref: user_agent.synthetic.type
+        requirement_level: opt_in
```

Comment:
oh, it has to be `opt_in` since it's an experimental attribute on a stable convention. We should be able to add comment to opt_in with the next weaver release.